pomerium/ctrl/bootstrap.go

```diff
@@ -26,6 +26,7 @@ func Apply(ctx context.Context, dst *config.Options, src *model.Config) error {
 		fn   func(context.Context, *config.Options, *model.Config) error
 	}{
 		{"secrets", applySecrets},
+		{"ca", applyCertificateAuthority},
```

Comment:
bootstrap config surface was intentionally kept to bare minimum - is there a special reason CA should be part of it and not set via the databroker config like other options?

Reply:
sorry I don't understand the distinction here. I thought this was where I needed to set options. I don't think I understand how to make this change.

Reply:
There are two reconcilers running watching for changes in the CRD. 

One is controlling core pomerium (the `pomerium/ctrl`) and makes a bare minimum config to keep it up and running and accepting gRPC databroker parameters. 

The other one sets full set of parameters via the databroker config - see https://github.com/pomerium/ingress-controller/blob/main/pomerium/config.go - my suggestion is to move handling of this CA there.